File: VB/NetCore/WhatIsMyBrowser.CommonTypesCoreVB/UserAgentParseDataResponse.vb
Imports Newtonsoft.Json

Public Class UserAgentParseDataResponse
    Inherits BaseResponse

    <JsonProperty("parse")>
    Public Property Parse As UserAgentParseResponseParse

    <JsonProperty("sanitization")>
    Public Property Sanitization As Sanitization

    <JsonProperty("version_check")>
    Public Property VersionCheck As VersionCheck
End Class

File: CS/NET40/WhatIsMyBrowser.CommonTypes/UserAgentParseDataResponse.cs
using Newtonsoft.Json;

namespace WhatIsMyBrowser.CommonTypes
{
    public class UserAgentParseDataResponse : BaseResponse
    {
        [JsonProperty("parse")]
        public UserAgentParseResponseParse Parse { get; set; }

        [JsonProperty("sanitization")]
        public Sanitization Sanitization { get; set; }

        [JsonProperty("version_check")]
        public VersionCheck VersionCheck { get; set; }
    }
}